resources/zib/terminology/CodeSystem-v3-RoleCode.supplement.xml

```diff
@@ -0,0 +1,44 @@
+<CodeSystem xmlns="http://hl7.org/fhir">

```

Comment:
Why has this CodeSystem been added in the context of this PR?

Reply:
That was a mistake, I removed it.